projects/fio/project.yaml

```diff
@@ -0,0 +1,10 @@
+homepage: "http://git.kernel.dk/fio.git"
+language: c++
+primary_contact: "axboe@kernel.dk"
+auto_ccs:
+- "p.antoine@catenacyber.fr"
```

Comment:
right indent these by 2, same for below.

Reply:
ok